UIforETW/UIforETWDlg.cpp

```diff
@@ -244,96 +1079,58 @@ BOOL CUIforETWDlg::OnInitDialog()
 {
 	CDialogEx::OnInitDialog();
 
-	// Load the F2 (rename) and ESC (silently swallow ESC) accelerators
-	hAccelTable_ = LoadAccelerators(AfxGetInstanceHandle(), MAKEINTRESOURCE(IDR_ACCELERATORS));
-	// Load the Enter accelerator for exiting renaming.
-	hRenameAccelTable_ = LoadAccelerators(AfxGetInstanceHandle(), MAKEINTRESOURCE(IDR_RENAMEACCELERATORS));
-	// Load the accelerators for when editing trace notes.
-	hNotesAccelTable_ = LoadAccelerators(AfxGetInstanceHandle(), MAKEINTRESOURCE(IDR_NOTESACCELERATORS));
-	// Load the accelerators for when the trace list is active.
-	hTracesAccelTable_ = LoadAccelerators(AfxGetInstanceHandle(), MAKEINTRESOURCE(IDR_TRACESACCELERATORS));
+	ATLTRACE2(L"Initializing dialog box!\r\n");
+
+	const HINSTANCE instanceHandle = AfxGetInstanceHandle();
+
+	hAccelTable_       = loadAcceleratorsForF2andESC(instanceHandle);
+	hRenameAccelTable_ = loadAcceleratorsForExitingRenaming(instanceHandle);
+	hNotesAccelTable_  = loadAcceleratorsForEditingTraceNotes(instanceHandle);
+	hTracesAccelTable_ = loadAcceleratorsForActiveTraceList(instanceHandle);
+
+	CRect windowRect = GetWindowRectFromHwnd(m_hWnd);
 
-	CRect windowRect;
-	GetWindowRect(&windowRect);
-	initialWidth_ = lastWidth_ = windowRect.Width();
-	initialHeight_ = lastHeight_ = windowRect.Height();
+	initialWidth_ = windowRect.Width();

```

Comment:
I'm not saying that this alternate form is worse, but is it clearly better? Churn has to be justified.